index_search_test.go

```diff
@@ -2,6 +2,7 @@ package meilisearch
 
 import (
 	"encoding/json"
+	"github.com/stretchr/testify/assert"
```

Comment:
Instead of using another import you should follow the convention in this file:

https://github.com/meilisearch/meilisearch-go/blob/main/index_search_test.go#L119